peerpod-ctrl/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/confidential-containers/cloud-api-adaptor/peerpod-ctrl
 go 1.19
 
 require (
-	github.com/confidential-containers/cloud-api-adaptor v0.0.1-alpha
+	github.com/confidential-containers/cloud-api-adaptor v0.0.1-alpha.0.20230329054600-4aee262d04f9
```

Comment:
Is this an intentional change?

I just want to confirm, since you reverted the similar change in #755.

https://github.com/confidential-containers/cloud-api-adaptor/compare/f4023a90c17710ee12b981a7de9421a834da0098..1a493977518744cd6f7629b05822181bcc449c1e

Reply:
that's odd, i didn't touch this file there, only the ../go.mod
(I had this commit there but i moved it here, but it was before your comment)

Reply:
OK, I see.